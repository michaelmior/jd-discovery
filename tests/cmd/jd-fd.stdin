{"Student ID":1234,"Semester":6,"Lecture":"Numerical Methods","TA":"John"}
{"Student ID":1221,"Semester":4,"Lecture":"Numerical Methods","TA":"Smith"}
{"Student ID":1234,"Semester":6,"Lecture":"Visual Computing","TA":"Bob"}
{"Student ID":1201,"Semester":2,"Lecture":"Numerical Methods","TA":"Peter"}
{"Student ID":1201,"Semester":2,"Lecture":"Physics II","TA":"Simon"}
