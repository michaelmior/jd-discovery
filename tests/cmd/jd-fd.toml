bin.name = "jd"
args = ["fd"]
