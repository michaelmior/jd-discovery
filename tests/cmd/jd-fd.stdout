["TA"] -> Lecture
["TA"] -> Student ID
["TA"] -> Semester
["Semester"] -> Student ID
["Student ID"] -> Semester
["Lecture", "Student ID"] -> TA
["Lecture", "Semester"] -> TA
